docs/building-a-form/creating-custom-fields-and-widgets.md

```diff
@@ -17,20 +17,20 @@ rjsf passes all field and widget components to `SchemaField` (and most other com
 - `ObjectField`
 - `ArrayField`
 - `FieldTemplate`
-- `TextWidget`
-- `SelectWidget`
-- `EmailWidget`
-- `CheckboxWidget`
-- `RadioWidget`
+- [`TextWidget`](available-widgets.md#textwidget)
+- [`SelectWidget`](available-widgets.md#selectwidget)
+- [`EmailWidget`](available-widgets.md#emailwidget)
+- [`CheckboxWidget`](available-widgets.md#checkboxwidget)
+- [`RadioWidget`](available-widgets.md#radiowidget)
 - `TextareaWidget`
 
 The us-forms-system uses these custom fields and widgets:
 
-- `YesNoWidget`
+- [`YesNoWidget`](available-widgets.md#yesnowidget)
 - `AddressField`
```

Comment:
Do we need this? Not sure what it's referring to. There's a `definition` for address fields, but it doesn't seem to fit with this list, because these are all widgets.

Reply:
I can remove this if it's incorrect... it was in the docs already. Same goes for everything else here; it's all just taken from the legacy docs.

Reply:
Yep, I figured! All the work you're doing on reorganizing content is helping bring to light existing issues in the docs, so it's super helpful :)